readme.md

```diff
@@ -82,6 +82,10 @@ mixpanel.people.clear_charges('billybob');
 // delete a user
 mixpanel.people.delete_user('billybob');
 
+// delete a user in Mixpanel Engage without altering $last_seen
+// - pass option `$ignore_time: true` to prevent the $last_seen property from being updated
+mixpanel.people.delete_user('billybob', {$ignore_time: true});
```

Comment:
This is very important if you ever want to re-import the same user. The $last_seen property will see the time of the delete_user event, unless $ignore_time is passed in.

If this isn't passed in, and the user profile is re-imported with the same uid, the $last_seen will see the delete_user event as the last action, and it's irreversible.